Dotnet/SAP/README.md

```diff
@@ -0,0 +1,13 @@
+# Tizen .NET Companion App Samples
+Tizen .NET companion app samples using Samsung Accessory Protocol(SAP).
+
+Open one of the following solutions and build it using [Visual Studio](https://www.visualstudio.com/downloads/) with [Visual Studio Tools for Tizen](https://developer.tizen.org/development/visual-studio-tools-tizen/installing-visual-studio-tools-tizen).
```

Comment:
Could you plz check whether this link is right? For my understanding, recently document team is migrating documents from developer.tizen.org to docs.tizen.org.

Reply:
I just received a reply from the document team that the URL link will be maintained.